src/api/handlers.js

```diff
@@ -0,0 +1,17 @@
+var req_validations = require("./validations.js").request_validations()
+
+module.exports.request_handlers = function () {
+	return {
+		get_hypothesis_res: function (hypothesis_id) {
+			return new Promise(function (resolve, reject) {
+				// Validate Params
+				var valid = req_validations.validate_hypothesis(hypothesis_id);
```

Comment:
probably call validate_hypothesis_id or only validate_uuid for now?
We can actually move that to hypothesis-core later on

Reply:
Then should I remove the utils file or keep it?